modules/040-node-manager/templates/node-group/_static_or_hybrid_script.tpl

```diff
@@ -75,5 +75,10 @@ chmod 0600 /var/lib/bashible/bootstrap-token
 touch /var/lib/bashible/first_run
 {{- end }}
 
-/var/lib/bashible/bootstrap.sh
+checkBashible=$(systemctl is-active bashible.timer)
+if [[ "$checkBashible" == "inactive" ]]; then
```

Comment:
Can do it in one line
```
if systemctl is-active -q bashible.timer; then
```

Reply:
In this case, the bootstrap.sh script will be executed on nodes under bashible, but not on the nodes, where bashible isn't configured.